cylc/flow/network/schema.py

```diff
@@ -703,6 +703,39 @@ class Meta:
     pruned = Boolean()
 
 
+class Runtime(ObjectType):
+    class Meta:
+        description = """
+Subset of runtime fields.
+
+Existing on 3 different node types:
+- Task/family Definition
+- Task/family cycle instance
+- Job
+
+Definition is the task/family hierarchy processed base.
+Task/famlily cycle instance is the broadcast modified (if any) of the base.
+Job is a record of what was run for a particular submit.
```

Comment:
I think this is clearer:
```suggestion
- Task/family definition (from the flow.cylc file)
- Task/family cycle instance (includes any broadcasts on top of the definition)
- Job (a record of what was run for a particular submit)
```